app/models/paging_schedule.rb

```diff
@@ -4,25 +4,29 @@
 #  PagingScheduleEstimate is a module that takes a requested item and can
 #  read the paging schedule configuration and estimate a date of arrival.
 ###
-module PagingSchedule
-  mattr_accessor :schedule do
-    []
-  end
+class PagingSchedule
+  class_attribute :schedule, default: []
+
+  SAL3_COORDINATES = %w[
```

Comment:
I'm confused why we're hard-coding this list. As noted on the ticket, there's an attribute in the location details we should be referencing.

Reply:
I'm not sure what you mean by "location details" can you help point me in the right direction?

Reply:
FOLIO locations have details. We've used them extensively for paging and scanning information for location-specific behaviors.

E.g. https://github.com/sul-dlss/sul-requests/blob/main/config/folio/locations.json#L312 (linked to for convenience. In practice, I think we should pull live data from graphql like we are all the other details)

We consume this as part of the graphql query:
https://github.com/sul-dlss/sul-requests/blob/main/app/services/folio_graphql_client.rb#L115-L125

The schema is tracked our graphql repo:
https://github.com/sul-dlss/folio-graphql/blob/main/src/schema.graphql#L4058-L4066

Last week, you merged the PR that registered the `pagingSchedule` attribute linked to this ticket:
https://github.com/sul-dlss/folio-graphql/pull/101

We need to populate FOLIO with this data (per the note on #1661) and consume it in this application.